pkgs/tools/networking/brook/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "brook";
+  version = "v20190401";
+  name = "${pname}-${version}";
```

Comment:
Please remove this line, it is redundant as `name` is automatically constructed from `pname` and `version`.

Reply:
Thanks for the review! `name` has been removed.